community/rethinkdb/APKBUILD

```diff
@@ -5,7 +5,7 @@ pkgver=2.3.6
 pkgrel=9
```

Comment:
Do we want to bump `pkgrel` for this change to compiled sources?

Reply:
Yep, I'll update